include/SFML/System/Vector2.inl

```diff
@@ -53,6 +53,138 @@ y(static_cast<T>(vector.y))
 }
 
 
+////////////////////////////////////////////////////////////
+template <typename T>
+T Vector2<T>::length() const
+{
+    static_assert(std::is_floating_point_v<T>, "Vector2::length() is only supported for floating point types");
+
+    using std::sqrt; // allow ADL
+
+    return sqrt(lengthSq());
+}
+
+
+////////////////////////////////////////////////////////////
+template <typename T>
+T Vector2<T>::lengthSq() const
+{
+    static_assert(std::is_floating_point_v<T>, "Vector2::lengthSq() is only supported for floating point types");
+
+    return this->dot(*this);
+}
+
+
+////////////////////////////////////////////////////////////
+template <typename T>
+Vector2<T> Vector2<T>::normalized() const
+{
+    static_assert(std::is_floating_point_v<T>, "Vector2::normalized() is only supported for floating point types");
+
+    assert(*this != Vector2<T>());
+    return (*this) / length();
```

Comment:
`operator*` has higher precedence than `operator/` so the extra parentheses could be removed if you wanted to.

Reply:
I had it without parentheses first, thought might add it for readability. I don't really mind, any way is good.

Reply:
I like the parentheses. I don't want to have to think about operator precedence when I read the code :)